SmartDeviceLink/public/SDLVoiceCommand.m

```diff
@@ -13,6 +13,7 @@
 @interface SDLVoiceCommand()
 
 @property (assign, nonatomic) UInt32 commandId;
+@property (copy, nonatomic, readwrite) NSArray<NSString *> *voiceCommands;
```

Comment:
When this line is removed, app crashes with code `BAD_ACCESS` since I am setting this property in `sdl_removeEmptyVoiceCommands` function of `SDLVoiceCommandManager`